try:
    print(10/2)
except Exception as e:
    print("error", e)    
finally:
    print("the code is running")  

print("we are learning a python ")      